docs/construct.rst

```diff
@@ -140,8 +140,31 @@ DOK arrays also support fancy indexing assignment if and only if all dimensions
    s[[0, 2], [2, 1], [0, 1]] = 5
    s[[0, 3], [0, 4], [0, 1]] = [1, 5]
 
-At the end, you can convert the :obj:`DOK` array to a :obj:`COO` array, and
-perform arithmetic or other operations on it.
+Alongside indexing assignment and retrieval, :obj:`DOK` arrays support any arbitrary broadcasting function
+to any number of arguments where the arguments can be :obj:`SparseArray` objects, :obj:`scipy.sparse.spmatrix`
+objects, or :obj:`numpy.ndarrays`. 
+
+.. code-block:: python
+
+   x = sparse.random((10, 10), 0.5, format="dok")
+   y = sparse.random((10, 10), 0.5, format="dok")
+   sparse.elemwise(np.add, x, y)
+
+:obj:`DOK` arrays also support standard `__ufunc__` operations and operators, including comparison operators,
+in combination with other objects implementing the `numpy` `ndarray.__array_ufunc__` method. For example,
+the following code will perform elementwise equality comparison on the two arrays
+and return a new boolean :obj:`DOK` array.
+
+.. code-block:: python
+
+   x = sparse.random((10, 10), 0.5, format="dok")
+   y = np.random.random((10, 10))
+   x == y
+
+:obj:`DOK` arrays are returned from elemwise functions and standard `__ufunc__` operations if and only if all 
```

Comment:
Same here.

Reply:
I changed it to just say universal... I'm not sure if there's a better technical term to refer to it by